tapeout/src/test/resources/macros/saed32.json

```diff
@@ -0,0 +1,186 @@
+[
+  {
+    "type": "sram",
+    "name": "SRAM1RW1024x8",
```

Comment:
The SAED stuff is not public - Synopsys keeps it locked up for use by its licensees only.  We've historically respected this for course materials (e.g., CS250) by keeping scripts behind the Berkeley wall.

That said, if it's already Google-compromised, then I guess I don't feel strongly about it.